python/graphstorm/config/argument.py

```diff
@@ -1545,6 +1569,10 @@ def _add_gnn_args(parser):
             help="The number of features in the hidden state")
     group.add_argument("--num-layers", type=int, default=argparse.SUPPRESS,
             help="number of layers in the GNN")
+    group.add_argument("--num-input-ngnn-layers", type=int, default=argparse.SUPPRESS,
+                       help="number of extra MLP layers in input layer.")
+    group.add_argument("--num-gnn-ngnn-layers", type=int, default=argparse.SUPPRESS,
+                       help="number of extra MLP layers between GNN layers.")
```

Comment:
you should also apply this to the decoders.

Reply:
Currently the performance on the decoders is not as good as what we do in the input and gnn layers. For the ngnn on the input and gnn layer, it has good performance on some datasets. I decide to delete the decoder part and check if it is necessary to add. If so, I would put that into another PR.

Reply:
overall, graphstorm needs to have very powerful decoders. currently, our decoders are just linear layers. i don't think it's enough.

Reply:
Agree